shub/utils.py

```diff
@@ -39,6 +39,24 @@
 STDOUT_ENCODING = sys.stdout.encoding or FALLBACK_ENCODING
 LAST_N_LOGS = 30
 
+_SETUP_PY_TEMPLATE = """\
```

Comment:
Creating setup.py from template was moved to `shub.utils` for similar reasons - it's used by `shub` and `shub.image` commands and keeping it in `shub.build` leads to import errors when 2 modules try to import each other.